autogen.sh

```diff
@@ -9,7 +9,9 @@ autoreconf --install --warnings=all --force
 
 args=""
 if [ -f "$topdir/.config.args" ]; then
-	args="$args $(cat $topdir/.config.args)"
+	args="$(cat $topdir/.config.args)"
```

Comment:
@krnowak thx for the previous input will fix it.

For putting config args files below rkt directory, don't know I'm not a fan, I mean if the user did run git clean -ffx then he wants to ignore the git ignore and other rules, so why bother ?

Having a single ignore .config.args file makes thing easy and everyone can put his own personal arguments there. Allowing to pass arbitrary files to autogen seems also ok, but I wouldn't bother about git clean -ffx and the path of the file.

Thoughts ?